Procfile

```diff
@@ -0,0 +1,2 @@
+web: gunicorn application:app
```

Comment:
We don't want to have a procfile. This should be removed.

Reply:
Done wait for commit

Reply:
Addressed in 
1286daf